client/src/stories/Component/NavBars/NavBar.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+import NavBarItem from "./NavBarItem";
+import PartisepantIcon from "../../assets/participantIcon24px.svg";
+import BillIcon from "../../assets/receipt24px.svg";
+import CourseIcon from "../../assets/course24px.svg";
+import Brand from "../Brand/Brand";
+import BrandLogo from "../../assets/logo.svg";
+
+function NavBar() {
+  return (
+    <Container>
+      <Brand imgSrc={BrandLogo} imgName="Firm logo" firmName="Kurs20k" />
+      <ul>
+        <NavBarItem
+          label="Teilnehmer"
+          href="www.google.de"
```

Comment:
I would also add the classical ```target='_blank'``` so that the user can click and open the file in a new tab.
Also, I would add the _**https://*_* thing before www.google.de and the other links

Reply:
Thanks for the hint with the target. I deliberately didn't use it, because it is not meant to be used to open a new tab here. I also want to give the user the decision if he wants to open the page in a new tab. Thanks for the hint. Google is only a placeholder and will be replaced as soon as the pages are ready.